cmd/podman/root.go

```diff
@@ -158,7 +158,7 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 
 	// Prep the engines
 	if _, err := registry.NewImageEngine(cmd, args); err != nil {
-		return err
+		return errors.Wrapf(err, "Cannot connect to the Podman socket, make sure their is a podman service running.")
```

Comment:
I'm not completely sold on the "REST API" bit I've added, but I think it brings clarity.  Another wording perhaps?
```suggestion
		return errors.Wrapf(err, "Cannot connect to the Podman socket, make sure there is a Podman REST API service running.")
```